guides/common/modules/snip_extract-directory-imported-project-server.adoc

```diff
@@ -0,0 +1 @@
+. On the importing {ProjectServer}, extract the `/var/lib/pulp/imports` directory to `/var/lib/pulp/`. You can empty the `/var/lib/pulp/imports` directory after a successful import.
```

Comment:
```suggestion
. On the importing {ProjectServer}, extract the `/var/lib/pulp/imports` directory to `/var/lib/pulp/`.
You can empty the `/var/lib/pulp/imports` directory after a successful import.
```
Please, one sentence per line.

Are you sure that the user has to do the extraction by themself? I thought that ProjectServer (?) does that during the import automatically. If so, please rephrase.